site/en/docs/devtools/accessibility/reference/index.md

```diff
@@ -185,50 +171,95 @@ View an element's computed accessibility properties in [the Accessibility pane][
 
 {% Img src="image/admin/KhrNElIocLRywp4Na7x7.png", alt="The Computed (Accessibility) Properties section.", width="800", height="506" %}
 
-**Figure 9**. The Computed (Accessibility) Properties section
-
-## View the contrast ratio of a text element in the Color Picker {: #contrast }
+## Make your website more readable {: #contrast }
```

Comment:
Should we move this part to a documentation itself (e.g. `[devtools/accessibility/fix-low-contrast)`?

Reply:
Yeah, I have a few ideas about that. I'll explain them in an email soonish.